src/seedu/addressbook/AddressBook.java

```diff
@@ -1153,6 +1186,15 @@ private static String getUsageInfoForExitCommand() {
     private static String removePrefixSign(String s, String sign) {
         return s.replace(sign, "");
     }
+    
+    /**
+     * Removes prefix in fullString
+     */
```

Comment:
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.